docs/learn-DOT.md

```diff
@@ -49,6 +53,59 @@ DOTs will be used to facilitate the consensus mechanism that underpins Polkadot.
 
 DOTs will have the ability to be bonded for a duration of time in order to add a new parachain to the network. The DOTs will be locked during their bonding period and will be released back to the account that bonded them after the duration of the bond has elapsed and the parachain is removed.
 
+### Vesting
+
+DOTs that are allocated to the Polkadot team or awarded to teams as part of 
+Web3 Foundation's grants program will have a vesting schedule attached. Vested
+balances are classified as a separate abstraction to the other locks that are
+placed on your balance. This means that although vested tokens cannot be transferred,
+they can still be used for voting in governance or staked in the staking system.
```

Comment:
Undeclared "this".

Nonetheless, the fact that vesting is a separate lock isn't what lets you use vesting funds in governance/staking, that's just a property of locks.